module/config/actor.mjs

```diff
@@ -137,6 +137,22 @@ export const TRAVEL_PACES = freezeEnum({
   }
 });
 
+/**
+ * Categories a language can (optionally) belong to
+ * @type {Record<string, string>}
+ */
+export const LANGUAGE_CATEGORIES = {};
```

Comment:
Let's put in an initial category, maybe something like "Spoken Languages" and we can put `common` inside it.